front-end/src/App.jsx

```diff
@@ -1,8 +1,26 @@
-import React from "react";
+import React, {useEffect} from "react";
 
 import ChartScreen from "./components/charts/chart-screen";
+import {useDispatch} from "react-redux";
+import {addData} from "./redux/slices/dataSlice";
 
 const App = () => {
+	const dispatch = useDispatch();
+	useEffect(() => {
+		fetch(
```

Comment:
this indentation is 8 spaces here; should be 2.

Reply:
@kaloyanpeev , let's configure prettier, so all these spaces and stuff are going to be solved.

Reply:
@Yavorss prettier should be configured already on my end, figured out what the problem was.

Reply:
There shoud be a prettier file. Check out https://prettier.io/docs/en/install.html

Reply:
Spaces were correctly configured, problem was that auto-save saved files to tabs